aequilibrae/distribution/gravity_application.py

```diff
@@ -142,20 +144,25 @@ def __init__(self, **kwargs):
         self.column_field = kwargs.get("column_field", None)
 
         self.impedance = kwargs.get("impedance")
-        self.model = kwargs.get("model")
+        self.model = kwargs.get("model")  # type: SyntheticGravityModel
         self.core_name = kwargs.get("output_core", "gravity")
-        self.output_name = kwargs.get("output", AequilibraeMatrix().random_name())
+        self.output_name = AequilibraeMatrix().random_name()
         self.nan_as_zero = kwargs.get("nan_as_zero", False)
         self.output = None
         self.gap = np.inf
         self.logger = logging.getLogger("aequilibrae")
+        self.procedure_date = ''
+        self.procedure_id = ''
+        self.__ipf = None  # type: Ipf
```

Comment:
perhaps newbie question here. The choice of using double underscore may cause issue (or help to not have issues) when inhereting the class. I am not sure what would be the case here.

Reply:
Not an issue